lib/core/mock/ecm-user.service.mock.ts

```diff
@@ -49,6 +49,17 @@ export let fakeEcmUser = {
     emailNotificationsEnabled: true
 };
 
+export let fakeEcmUser2 = {
```

Comment:
use `const` over `let`
```suggestion
export const fakeEcmUser2 = {
```